src/kixi/comms/components/kafka.clj

```diff
@@ -322,7 +322,8 @@
   (start [component]
     (if-not (:producer-in-ch component)
       (let [topics (or topics {:command "command" :event "event"})
-            origin (or origin (.. java.net.InetAddress getLocalHost getHostName))
+            origin (or origin (try (.. java.net.InetAddress getLocalHost getHostName)
+                                   (catch Throwable _ "<unknown>")))
```

Comment:
No, it just returns "<unknown>" as the host, which becomes `origin`. Really, people should be specifying the `origin` so this is like Plan C, assuming A and B have failed.